lib/internal/errors.js

```diff
@@ -229,6 +229,8 @@ E('ERR_NAPI_CONS_PROTOTYPE_OBJECT', 'Constructor.prototype must be an object');
 E('ERR_NO_CRYPTO', 'Node.js is not compiled with OpenSSL crypto support');
 E('ERR_NO_ICU', '%s is not supported on Node.js compiled without ICU');
 E('ERR_NO_LONGER_SUPPORTED', '%s is no longer supported');
+E('ERR_NOT_FUNCTION', '%s must be a function');
```

Comment:
I think all of the instances that use this error would be better if they used the existing `ERR_INVALID_ARG_TYPE`.